corehq/apps/locations/forms.py

```diff
@@ -96,6 +96,9 @@ def __init__(self, location, bound_data=None, is_new=False, user=None,
 
         self.custom_data = self.get_custom_data(bound_data, is_new)
 
+        self.custom_data.form.helper.label_class = 'col-sm-3 col-md-4 col-lg-2'
+        self.custom_data.form.helper.field_class = 'col-sm-4 col-md-5 col-lg-3'
```

Comment:
Actually, I'm mistaken - it's the "Default monthly consumption values" section that is broken.